manifests/init.pp

```diff
@@ -61,7 +61,16 @@
   Optional[Integer] $logconfig_file_max_log_size = undef,
   Optional[Integer] $logconfig_file_backups      = undef,
   Optional[String] $logconfig_file_category      = undef,
-) {
+
+  # Padoptions
+  Etherpad::Stpadoptions $padoptions = {},
+) inherits ::etherpad::params {
+  #Merged values provides by user and default values
+  if $padoptions {
```

Comment:
Is this `if` actually needed?  If the parameter hasn't been specified, I think merging the default empty hash would work?

Reply:
Fixed.

Reply:
Did you push your fix?

Reply:
In few minutes.

Reply:
I did it.